ggshield/core/config/user_config.py

```diff
@@ -63,14 +63,55 @@ def validate_policy_id(policy_id: str) -> bool:
     return bool(POLICY_ID_PATTERN.fullmatch(policy_id))
 
 
-def validate_policy_ids(values: Iterable[str]) -> None:
-    invalid_excluded_policies = [
-        policy_id for policy_id in values if not validate_policy_id(policy_id)
-    ]
-    if len(invalid_excluded_policies) > 0:
-        raise ValidationError(
-            f"The policies {invalid_excluded_policies} do not match the pattern '{POLICY_ID_PATTERN.pattern}'"
-        )
+@marshmallow_dataclass.dataclass
+class ConfigIgnoredElement(FilteredConfig):
```

Comment:
IMO, in this component, in case data is a dict, we should check the keys of the dict and display a warning if the dict has an unknown key

Reply:
Skipped for now, as other classes do not do this